src/ui/components/AllowanceView/AllowanceView.tsx

```diff
@@ -43,7 +43,8 @@ export function AllowanceView({
   const positionQuantity = useMemo(
     () =>
       positionsResponse?.positions.find(
-        (position) => position.chain === chain.toString()
+        (position) =>
+          position.chain === chain.toString() && position.dapp == null
```

Comment:
I have just checked how its done in the web app:
https://github.com/zeriontech/pulse-frontend/blob/bc2e800d76f49cb37e173912a7682d5df920eb33/src/Root/App/Main/tabs/Overview/Positions/Positions.tsx#L120-L133

https://github.com/zeriontech/pulse-frontend/blob/bc2e800d76f49cb37e173912a7682d5df920eb33/src/Root/App/Main/tabs/Overview/Positions/Positions.tsx#L106